src/libraries/Native/Unix/System.Globalization.Native/pal_icushim_static.c

```diff
@@ -30,6 +38,37 @@ static void U_CALLCONV icu_trace_data(const void* context, int32_t fnNumber, int
 #ifdef __EMSCRIPTEN__
 #include <emscripten.h>
 
+EMSCRIPTEN_KEEPALIVE const char* mono_wasm_get_icudt_name(const char* culture);
+
+EMSCRIPTEN_KEEPALIVE const char* mono_wasm_get_icudt_name(const char* culture)
+{
+    // Based on https://github.com/dotnet/icu/tree/maint/maint-67/icu-filters
+
+    // Use full one if culture is null or empty
+    if (!culture || strlen(culture) < 2)
```

Comment:
I don't get why we return icudt.dat when culture is false or its length is less than 2 in length. 

Shouldn't we have more logic to detect we're using `icudt.dat`?

Reply:
it's an API to get an icudt*.dat name from the ICU nuget for a given culture.
`NULL` means "give me the fallback(full) one". Mono-Browser users are not forced to use this API if they have their own icudt.dat file.

Reply:
Ah ok. Sounds good.